docs/install-upgrade/build-wiring.md

```diff
@@ -3,36 +3,93 @@
+    type: mclag
+```
+
+## Design Discussion
+This section is meant to help the reader understand how to assemble the primitives presented by the Fabric API into a functional fabric.
+
+### VPC
+
+A VPC allows for isolation at layer 3. This is the main building block for users when creating their architecture. Hosts inside of a VPC belong to the same broadcast domain and can communicate with each other. The hosts inside of a VPC will likely need to connect to other VPCs or the outside world. To communicate between two VPC a *peering* will need to be created. A VPC can be a logical separation of workloads. By separating these workloads additional controls are available. The logical separation doesn't have to be the traditional database, web, and compute layers it could be development teams who need isolation, it could tenants inside of an office building, or any separation that allows for better control of the network. Once your VPCs are decided, the rest of the fabric will come together. With the VPCs decided traffic can be prioritized, security can be put into place, and the wiring can begin. The fabric allows for the VPC to exist beyond a single switch, which gives flexibility when the physical world meets the digital.
+
+### Connection
+
+A connection represents the physical wires in your data center. They connect switches to other switches or switches to servers.
+
+#### Server Connections
+
+A server connection is a connection used to connect servers to the fabric. The fabric will configure the server-facing port according to the type of the connection (MLAG, Bundle, etc).The configuration of the actual server needs to be done by the server administrator. The server name is not validated by the fabric and is used as metadata to identify the connection. A server connection can be one of: 
+
+- *Unbundled* - A single cable connecting switch to server.
+- *Bundled* - Two or more cables going to a single switch, a LAG or similar.
+- *MCLAG* -  Two cables going to two different switches, also called dual homing. The switches will need a fabric link between them.
+- *ESLAG* - Two to four cables going to different switches, also called multi-homing. If four links are used there will need to be four switches connected to a single server with four NIC ports.
+
+#### Fabric Connections
+
+Fabric connections serve as connections between switches, they form the fabric of the network.
+
+
+### VPC Peering
+
+VPCs need VPC Peerings to talk to each other. VPC Peerings come in two varieties: local and remote.
+
+#### Local VPC Peering
+
+When there is no dedicated border/peering switch available in the fabric we can use local VPC peering. This kind of peering tries to send traffic between the two VPC's on the switch where either of the VPC's has workloads attached. Due to limitation in the Sonic network operating system this kind of peering bandwidth is limited to the number of VPC loopbacks you have selected while initializing the fabric.
```

Comment:
", in  this kind of peering, bandwidth is limited ..." ? If I understand the sentence...